example/postgres/index.js

```diff
@@ -0,0 +1,43 @@
+const express = require('express');
+const http = require('http');
+const { Client } = require('pg');
+const terminus = require('../../lib/terminus');
+const app = express();
+const client = new Client({
+  database: 'terminus',
+  user: 'test',
+  password: 'test'
+});
+
+app.get('/', async (req, res) => {
+  const result = await client.query('SELECT $1::text as message', ['Hello world!']);
+  res.send(result.rows[0].message);
+});
+
+function onHealthCheck() {
+  return Promise.resolve();
```

Comment:
just realized after merging, that for the healthCheck we should check the health of the db connection - would running a `SELECT 1` make sense here @lukaszewczak ? if so, can you send a fix for this?

Reply:
oh ok I will prepare it